pkg/controller/controller_instance.go

```diff
@@ -1347,12 +1369,18 @@ func (c *controller) pollServiceInstance(instance *v1beta1.ServiceInstance) erro
 					v1beta1.ConditionUnknown,
 					errorOrphanMitigationFailedReason,
 					"Orphan mitigation failed: "+s)
-			} else {
+			} else if deleting || provisioning {
```

Comment:
Note to @n3wscott this will conflict with https://github.com/kubernetes-incubator/service-catalog/pull/1509, easy merge and would have been helpful in this PR to just have to find and fix this once.